app/lib/core/models/initial_atr.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+
+@immutable
+class InitialAtr {
```

Comment:
- [ ] I think a better way to use this class would be to name it `AtrIdentifier`. Because it identifies the ATR, so then the `AnimalTransportRecord` model object will have this object in it instread of separate fields for `author` and `atrID`

Reply:
```
AnimalTransportRecord {
   final AtrIdentifier identifier;
   // ...
}
```

Reply:
What is your plan for the rest of the fields on your example? We need a class that will push specifically userId and isComplete fields initially. If you plan to add more here, I don't think it'll work.

Reply:
We would first use `InitialAtr`, then when we make the rest of the `ATR` we give it the `InitialAtr` and start adding the other fields as we go. That would mean that the ATR should have `Optional` ShipperInfo, etc. until the next section is added.

Reply:
We'll discuss the full workflow of creating a new ATR, editing it, saving it, repeating those two previous steps, and completing it, either in the next scrum or backlog refinement. 

This discussion goes beyond GitHub comments and is the result of confusion as to how code communicates between modules.